Makefile

```diff
@@ -0,0 +1,53 @@
+### Auto-generated files
+codegen-all: go-codegen js-codegen
+
+go-codegen:
+	go run superchain/internal/codegen/main.go
+
+js-codegen:
+	node ./scripts/codegen.js
+
+### Linting
+lint-all:
+	golangci-lint run superchain/... validation/... add-chain/... --fix
+
+### Testing
+test-all: test-add-chain test-superchain test-validation
+
+test-add-chain:
+# We separate the first test from the rest because it generates artifacts
+# Which need to exist before the remaining tests run.
+	go test ./add-chain/... -run TestAddChain_Main -v
+	go test ./add-chain/... -run '[^TestAddChain_Main]' -v
+	make clean-add-chain
+
+test-superchain:
+	go test ./superchain/... -v
+
+test-validation:
+	go test ./validation/... -v
+
+### Cleaning
+clean-add-chain:
+	rm -f superchain/configs/sepolia/testchain_*.yaml
+	rm -f superchain/extra/addresses/sepolia/testchain_*.json
+	rm -f superchain/extra/genesis-system-configs/sepolia/testchain_*.json
```

Comment:
Do we still need this?

Reply:
I think that one is handy to have around